### DISCUSSION TITLE
chatgpt plus 用户配置openai出错429: { "error": { "message": "You exceeded your current quota, please check your plan and billing details.", "type": "insufficient_quot

### DISCUSSION THREAD

**Discussion by nancyccw** [immersive-translate/immersive-translate#553] [Help] [answered]
### 概述 | Summary

API-key是对的。我重新生成了一个key。账户付费。GPT-4可用。
429: { "error": { "message": "You exceeded your current quota, please check your plan and billing details.", "type": "insufficient_quot

请教原因

**Comment by theowenyoung** [comment, marked as answer]
> check your plan and billing details

登录到账户页看看 usage情况？

**Reply by nancyccw** [reply]
在哪里看usage呢

**Reply by theowenyoung** [reply]
https://platform.openai.com/account/usage

**Reply by nancyccw** [reply]
确实是Expired了几天。谢谢提醒。但是我的付费是到6月8日。这个是怎么回事呢

**Reply by theowenyoung** [reply]
这个是openai的政策吧。

**Reply by nancyccw** [reply]
那需要等下次自动扣费才能正常使用吗

**Reply by theowenyoung** [reply]
不懂～ 我不太清楚你的情况，也许你可以咨询一下openai的客服？

**Reply by nancyccw** [reply]
好的。谢谢
